packages/create-contentful-app/README.md

```diff
@@ -0,0 +1,19 @@
+# Create Contentful App
+
+This project will make it easy to bootstrap Contentful Apps, with React,
```

Comment:
"to bootstrap a Contentful App using libraries like x, y, z"

Reply:
Add links to "Contentful Apps" and library names. Also use correct library names (for example Forma is "Forma 36").